flocker/volume/_ipc.py

```diff
@@ -79,7 +79,7 @@ def acquire(volume):
         :param Volume volume: The volume which will be acquired by the
             remote volume manager.
 
-        :return: The UUID of the remote volume manager (as ``unicode``).
+        :return: The node ID of the remote volume manager (as ``unicode``).
```

Comment:
I wonder if (not now) it would make sense to represent UUIDs as UUID instances instead of unicode.